gcp-bundle/LICENSE

```diff
@@ -277,11 +297,75 @@ License: Apache-2.0 - https://www.apache.org/licenses/LICENSE-2.0.txt
 
 --------------------------------------------------------------------------------
 
+Group: com.google.api.grpc  Name: grpc-google-cloud-bigquerystorage-v1  Version: 3.14.0
+Project URL: https://github.com/googleapis/java-bigquerystorage/grpc-google-cloud-bigquerystorage-v1
```

Comment:
The project URL is not correct here.

You should use https://github.com/googleapis/java-bigquerystorage/ or https://github.com/googleapis/java-bigquerystorage/tree/main/proto-google-cloud-bigquerystorage-v1